docs/cow-protocol/reference/contracts/periphery/README.mdx

```diff
@@ -0,0 +1,17 @@
+# Periphery
+
+Peripheral contracts are those that are not necessary for CoW Protocol to function, but are used to enhance user experience.
+
+## Deployments
+
+:::tip
+
+CoW Protocol contracts are deployed using deterministic addresses. This means that the same contract code will always be deployed to the same address. This is done by using the `CREATE2` opcode.
```

Comment:
This is in general not true! For example, the ETH flow contract was considered disposable and deployed with standard `CREATE`. The main reason why we didn't make it deterministic is that it needs to know the WETH address on deployment, which is different on each network and is part of the deployment coode (barring gas-expensive tricks to avoid that).